File: 프로그래머스/lv0/120835. 진료 순서 정하기/진료 순서 정하기.py
def solution(emergency):
    answer = [1]*len(emergency)
    for i in range(len(emergency)-1):
        for j in range(i,len(emergency)):
            if emergency[i] > emergency[j]:
                answer[j] +=1
            elif emergency[i] < emergency[j]:
                answer[i]+=1
    return answer

File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.java
class Solution {
    public int[] solution(int[] emergency) {
        int[] answer = new int[emergency.length];

        for(int i=0; i<emergency.length; i++){
            for(int j=i; j<emergency.length; j++){
                if(emergency[j]>emergency[i]){
                    answer[i]+=1;
                }else{
                    answer[j]+=1;
                }
            }
        }
        return answer;
    }
}